nx-X11/programs/Xserver/hw/nxagent/Args.c

```diff
@@ -1351,7 +1351,7 @@ static void nxagentParseOptions(char *name, char *value)
     errno = 0;
     sleep_parse = strtol(value, NULL, 10);
 
-    if ((errno) && (0 == sleep_parse))
+    if ((errno) && (sleep_parse == 0))
```

Comment:
@Ionic: I noticed these comparisons, too, when you worked on the sleep and tolerance feature. Any reason for having the order <val> == <var>?

Reply:
I think this is a coding habit that prevents acciently writing `=` instead of `==`. The compiler will complain if you try to assign a value to a constant.

Reply:
Yes, exactly for this reason. It's a pretty common pattern to avoid mistakes (and I'd like to keep that part.)